profiler/include/profiler/profile_transpose_impl.hpp

```diff
@@ -153,10 +153,6 @@ bool profile_transpose_impl(int do_verification,
             std::cout << "Perf: " << std::setw(10) << ave_time << " ms, " << tflops << " TFlops, "
                       << gb_per_sec << " GB/s, " << op_name << std::endl;
 
-            // pass = pass & ck::utils::check_err(b_device_result, b_host_result);

```

Comment:
How result is verified?

Reply:
The result is verified in line 127

Reply:
You're right. Just to make sure. Why kernel is run twice? Kernel uses some atomic adds so the first one is for check the result and second one for measure the performance? If yes, could You put some comment?